apps/client/src/app/shop/[shop_id]/_components/ShopProduct/ShopProduct.tsx

```diff
@@ -0,0 +1,47 @@
+import Link from "next/link";
+import { Box, Flex, Text } from "@chakra-ui/react";
+
+import { Icon, ImageBox } from "~/components";
+
+import type { GetProductsByShopResponse } from "../../_types";
+
+export const ShopProduct = (productInfo: GetProductsByShopResponse["result"]["productList"][number]) => {
+  const { id, name, topCategory, introduce, price } = productInfo;
+
+  return (
+    <Link href={`/shop/${id}/products`} style={{ position: "relative" }}>
+      <Box position="absolute" top="8px" right="8px">
+        <Icon name="sell" />
+      </Box>
+      <ImageBox
+        src="https://picsum.photos/200"
+        alt="ProductImage"
+        boxSize="172px"
+        borderRadius="8px"
+        imgOptions={{ priority: true }}
+      />
+      <Flex align="baseline" columnGap="4px" mt="24px">
+        <Text fontSize="16px" fontWeight="700" w="auto" overflow="hidden" textOverflow="ellipsis" whiteSpace="nowrap">
+          {name}
+        </Text>
+        <Text fontSize="12px" fontWeight="500" w="full">
+          {topCategory}
+        </Text>
+      </Flex>
+      <Text
+        mt="8px"
+        mb="16px"
+        fontSize="14px"
+        fontWeight="300"
+        overflow="hidden"
+        textOverflow="ellipsis"
+        whiteSpace="nowrap"
+      >
+        {introduce}
+      </Text>
+      <Text fontSize="14px" fontWeight="500">
+        {price}원 ~
```

Comment:
price값이 number 타입으로 응답되는 것으로 알고 있는데 화면에서는 3자리 숫자마다 콤마를 찍어 표기해줘야 할 것 같습니다. 확인 한 번 부탁드립니다. 😄 

```suggestion
        {price.toLocaleString()}원 ~
```

Reply:
@showme0241 
[feat: 콤마 코드 추가](https://github.com/shophubkr/shophub-client/pull/52/commits/ec47c4adf77fac16da7a77ddd8dcb28774ff3100)
확인하고 수정했습니다!